channels/rdpdr/client/devman.c

```diff
@@ -49,6 +53,9 @@ DEVMAN* devman_new(rdpdrPlugin* rdpdr)
 {
 	DEVMAN* devman;
 
+	if (!rdpdr)
```

Comment:
Just curious, is there any reason why rdpdr would be NULL ? 
I don't like code that protect against itself, because you feel suspicious about everything. If callers are expected to not provide a NULL pointer, let's adjust the callers...